tests/integration/helpers/cluster.py

```diff
@@ -778,9 +792,12 @@ def add_zookeeper_startup_command(self, command):
             - {instance_config_dir}:/etc/clickhouse-server/
             - {db_dir}:/var/lib/clickhouse/
             - {logs_dir}:/var/log/clickhouse-server/
+            - /etc/passwd:/etc/passwd:ro
```

Comment:
@qoega any CH integration test should be runnable by non-privileged user. 
In the container we have UID same as at host box (because of user: '{user}' in docker compose, generated y cluster.py), while /etc/password record is missed. This is strange but Ok for CH, while it does not work for Kerberos.
That is why it is not feasible to have static /etc/password, is must contain record for a user that run the test.
As far as I know, passing /etc/password from host is a rather common trick.
It is a good point about MAC and Windows. kerberaized kafka is disabled for a non-Linux build. Should we somehow limit the test for Linux?

Reply:
Regarding the original concern about /etc/passwd on Mac or Windows host, actually Docker host is a Linux anyway. Mac uses LinuxKit VM, Windows something similar.
It is common to rely on it in many cases, e.g. substituting /dev/random.

Reply:
One of the alternative workarounds - is to create the user internally (you can do useradd inside docker with needed uid).

Reply:
> One of the alternative workarounds - is to create the user internally (you can do useradd inside docker with needed uid).

Sure, it is possible to run container as root then create user with specified UID then switch to the user (somehow).
I just tried to be as minimally invasive as possible.